t/ffi/variadic.c

```diff
@@ -2,22 +2,72 @@
 #ifdef FFI_PL_PROBE_VARIADIC
 #include <stdio.h>
 #include <stdarg.h>
+#include "libtest.h"
 
-int
+EXTERN int
```

Comment:
This also fixes a bug where the variadic functions weren't being tested on Visual C++ because the tester functions for it weren't being exported from the DLL :/